operator/pkg/model/ingestion/gateway_test.go

```diff
@@ -34,6 +34,14 @@ var basicHTTP = Input{
 					Protocol: "HTTP",
 				},
 			},
+			Infrastructure: &gatewayv1beta1.GatewayInfrastructure{
+				Labels: map[gatewayv1beta1.AnnotationKey]gatewayv1beta1.AnnotationValue{
+					"internal-loadbalancer-label": "true",
+				},
+				Annotations: map[gatewayv1beta1.AnnotationKey]gatewayv1beta1.AnnotationValue{
+					"internal-loadbalancer-annotation": "true",
+				},
```

Comment:
@sayboras I believe this conflicted with the `s/gatewayv1beta1/gatewayv1` rename in https://github.com/cilium/cilium/pull/29115 :(.

Reply:
Fix-up coming via https://github.com/cilium/cilium/pull/29143.

Reply:
oh sorry :(